src/connection/sql_connection.go

```diff
@@ -70,6 +73,16 @@ func CreateConnectionURL(args *args.ArgumentList) string {
 	query.Add("dial timeout", args.Timeout)
 	query.Add("connection timeout", args.Timeout)
 
+	if args.ExtraConnectionURLArgs != "" {
+		mockURL, err := url.Parse("http://example.com?" + args.ExtraConnectionURLArgs)
+		if err == nil {
```

Comment:
why is this condition ? could you simplify the logic and use just url.ParseQuery (checking the output error if any)